doc/tutorials/using_sbl_on_up2.rst

```diff
@@ -0,0 +1,292 @@
+   the "Version" argument, and we can set ``"Version": 26000`` instead 
+   of ``"Version": 26880`` for example.
+
+Clone the source code of ``acrn-hypervisor`` and build SOS and LaaG image:
+
+.. code-block:: none
+   
+   cd ~
+   git clone https://github.com/projectacrn/acrn-hypervisor
+   sudo ./acrn-hypervisor/devicemodel/samples/up2/create-up2-images.sh --images-type all --clearlinux-version 26880 --laag-json uos.json --acrn-code-path ~/acrn-hypervisor/
+
+
+This step will generate the images of SOS and LaaG:
+
+* sos_boot.img
+* sos_rootfs.img
+* up2_laag.img
+
+.. note::
+   When building images, you can modify the ``--clearlinux-version`` argument 
+   to a specific version (such as 26800). To generate the images of SOS only, 
+   modify the ``--images-type`` argument to ``sos``.
+   
+
+We still need the binary image for GPT partitions and 
+configuration file for flashing:
+
+* partition_desc.bin
+* flash_LaaG.json
+
+.. note::
+   ``partition_desc.bin`` and ``flash_LaaG.json`` are in the diretory 
+   ``~/acrn-hypervisor/doc/tutorials/``.
+
+
+.. table::
+      :widths: auto
+
+      +------------------------------+---------------------------------------------------+
+      | Filename                     | Description                                       |
+      +==============================+===================================================+
+      | sos_boot.img                 | This SOS image contains the ACRN hypervisor and   |
+      |                              | SOS kernel.                                       |
+      +------------------------------+---------------------------------------------------+
+      | sos_rootfs.img               | This is the root filesystem image for the SOS. it |
+      |                              | contains the Device Models implementation and     |
+      |                              | SOS user space.                                   |
+      +------------------------------+---------------------------------------------------+
+      | partition_desc.bin           | This is the binary image for GPT partitions       |
+      +------------------------------+---------------------------------------------------+
+      | up2_laag.img                 | This is the root filesystem image for the SOS.    |
+      |                              | It has an integrated kernel and userspace.        |
+      +------------------------------+---------------------------------------------------+
+      | flash_LaaG.json              | Configuration file for Intel® Platform Flash Tool |
+      |                              | to flash SOS image + hypervisor/SOS boot image +  |
+      |                              | SOS userland                                      |
+      +------------------------------+---------------------------------------------------+
+
+.. note::
+   In this step, please build SOS and LaaG images in Clear Linux rather than Ubuntu. 
+
+Download and install flash tool
```

Comment:
I was looking for a webpage that can be used as a general intro to the Intel Platform Flash Tool Lite but the best I found was this: https://github.com/projectceladon/tools. Can you add a link to this as, despite not being a proper page, it has the merit of detailing which host OS are supported (useful if someone does not want to use Ubuntu for example).

Reply:
The following download link of "Intel® Platform Flash Tool Lite" is https://github.com/projectceladon/tools/tree/master/platform_flash_tool_lite/latest.